docs: update faq with new questions, fix menus (#5924)
ddev/ddev: docs/content/users/topics/index.md

# Hosting, Deployment, and Sharing

> DDEV does not provide capabilities for deploying your code, as it's focused on being a **local development** solution. Teams almost universally use Git to manage their code and some Git provider like GitHub or GitLab to store it. They use many different types of deployment tools to actually get the code to their upstream servers or integration environments. But for local development, DDEV provides capabilities to pull upstream databases and user-generated files (and to push them in some circumstances).

Reviewer: ```suggestion
DDEV does not provide capabilities for deploying your code, as it's focused on being a **local development** solution. Teams almost universally use Git to manage their code and some Git provider like GitHub or GitLab to store it. They use many different types of deployment tools to actually get the code to their upstream servers or integration environments. For local development, DDEV provides capabilities to pull upstream databases and user-generated files as well as push those in some circumstances.
```
the only nitpick might be trying to avoid the parenthesis at the end? perhaps like that?
